mantis-common/build.gradle

```diff
@@ -24,11 +24,11 @@ ext {
 
```

Comment:
the `ext` variable can be safely removed with this change.

Reply:
yep, that'd be desired to realize the full benefits of this work. 
The different versions would make it tricky and here's a good starting tip:
1. For major version difference, Create a new library variable.
2. For minor and patch, try to bump to the latest version for all subprojects (one variable per library and see if builds correctly).